kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/OperatorChangeDetectionST.java

```diff
@@ -70,12 +71,13 @@ void shouldUpdateDeploymentWhenKafkaProxyIngressChanges(String namespace) {
         LOGGER.info("Kafka proxy ingress edited");
 
         // Then
-        await().atMost(Duration.ofSeconds(90)).untilAsserted(() -> {
-            Deployment proxyDeployment = kubeClient.getDeployment(namespace, "simple");
-            assertThat(proxyDeployment).isNotNull();
-            OperatorAssertions.assertThat(proxyDeployment.getSpec().getTemplate().getMetadata()).hasAnnotationSatisfying("kroxylicious.io/referent-checksum",
-                    value -> assertThat(value).isNotEqualTo(originalChecksum));
-        });
+        var deployment = await().atMost(Duration.ofSeconds(90))
```

Comment:
What's the thinking behind the refactoring?  My gut feel is that there is an operator bug rather than a test bug.

Reply:
Personally, I'd exclude this change from the scope of this PR.  Let's work out what's wrong with OCDST separately.    We can merge this PR with that test failure (as it is unrelated).

Reply:
Just checking if the change works on Jenkins. Locally it worked, but still failing here.